tests/foreman/ui/test_activationkey.py

```diff
@@ -55,14 +55,14 @@ def setUpClass(cls):
         super(ActivationKey, cls).setUpClass()
 
     def create_sync_custom_repo(self, product_name=None, repo_name=None,
-                                repo_url=None, repo_type=None):
+                                repo_url=None, repo_type=None, org_id=None):
         """Create product/repo, sync it and returns repo_id"""
         product_name = product_name or gen_string("alpha", 8)
         repo_name = repo_name or gen_string("alpha", 8)
         # Creates new product and repository via API's
         product_attrs = entities.Product(
             name=product_name,
-            organization=self.org_id
+            organization=org_id or self.org_id
```

Comment:
@kbidarkar all path creating is done using string format, like https://github.com/SatelliteQE/robottelo/blob/master/robottelo/entities.py#L88. With that said, format will automatically convert to string.